core/jvm/src/test/scala/scalaz/zio/RTSSpec.scala

```diff
@@ -548,7 +548,7 @@ class RTSSpec(implicit ee: ExecutionEnv) extends AbstractRTSSpec with AroundTime
     unsafeRun(IO.point(42).race(IO.never)) must_=== 42
 
   def testRaceOfFailNever =
-    unsafeRun(IO.fail(24).race(IO.never).timeout[Option[Int]](None)(Option.apply)(10.milliseconds)) must beNone
+    unsafeRun(IO.fail(24).race(IO.never).timeout0[Option[Int]](None)(Option.apply)(10.milliseconds)) must beNone
```

Comment:
Same, `IO.fail(24).race(IO.never).timeout(10.milliseconds)`

Reply:
I changed it.